Source/VideoProgressView.swift

```diff
@@ -0,0 +1,119 @@
+import UIKit
+
+class VideoProgressView: UIView {
+
+    static let Height = CGFloat(55.0)
+    private static let ProgressBarXMargin = CGFloat(65.0)
+    private static let ProgressBarYMargin = CGFloat(23.0)
+    private static let ProgressBarHeight = CGFloat(6.0)
+
+    private static let TextLabelWidth = CGFloat(36.0)
+    private static let TextLabelHeight = CGFloat(18.0)
+    private static let TextLabelMargin = CGFloat(18.0)
+
+    var duration = 0.0 {
+        didSet {
+            if self.duration != oldValue {
+              self.durationTimeLabel.text = self.timeStringForSeconds(self.duration)
+            }
+        }
+    }
+    var currentTime = 0.0 {
+        didSet {
+            self.currentTimeLabel.text = self.timeStringForSeconds(self.currentTime)
+            self.setFrameForProgressBar()
+        }
+    }
+
+    var progressPercentage : Double {
```

Comment:
Please update so the `:` follows the variable name, there are another few places that might also need this change.

Preferred:

```swift
var progressPercentage: Double {
```